zk_toolbox/crates/zk_inception/src/commands/ecosystem/create.rs

```diff
@@ -114,3 +119,19 @@ fn create(args: EcosystemCreateArgs, shell: &Shell) -> anyhow::Result<()> {
     logger::outro(msg_created_ecosystem(ecosystem_name));
     Ok(())
 }
+
+fn check_link_to_code(path: &Path) -> anyhow::Result<()> {
+    if !path.exists() {
+        bail!(msg_path_to_zksync_does_not_exist_err(
+            path.to_str().unwrap()
+        ));
+    }
+
+    if !path.join("Cargo.toml").exists() {
+        bail!(msg_directory_does_not_contain_cargo_toml_err(
+            path.to_str().unwrap()
+        ));
+    }
```

Comment:
it's a bit of a weird check, any rust directory will pass it

Reply:
You could try `git remote -v | grep matter-labs/zksync-era`

And if this check doesn't pass, we could add a question about adding it forcefully. E.g. this use is absolutely sure what is he doing